uxpin.config.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
         include: [
           'src/components/Alert/Alert.js',
           'src/components/Page-Title/Page-Title.js',
+          'src/components/Link/Link.js',
```

Comment:
Let's move this down to keep the proper alphabetical order, please

Reply:
changes added